src/SFA.DAS.ApprenticeAccounts.UnitTests/MockDbSetSetup.cs

```diff
@@ -0,0 +1,21 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Moq;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace SFA.DAS.ApprenticeAccounts.UnitTests;
+public static class MockDbSetSetup
+{
+    public static Mock<DbSet<T>> CreateQueryableMockDbSet<T>(IEnumerable<T> data) where T : class
```

Comment:
This is a leaky abstraction, FYI we do not do this any where else, but it is ok to follow the suite here.